build/build-push-bundles.sh

```diff
@@ -1,27 +1,49 @@
 #!/bin/bash
 
-set -e
+set -ex
+
+CONTAINER_TOOL="${CONTAINER_TOOL:-podman}"
 
 BASE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." >/dev/null 2>&1 && pwd)"
 
+if [[ "${CONTAINER_TOOL}" == "podman" ]]; then
+    host_platform="$(podman info --format '{{.Version.OsArch}}')"
+else
+    host_platform="$(docker info --format '{{.ClientInfo.Os}}/{{.ClientInfo.Arch}}')"
+fi
+
 for dirpath in $(find ${BASE_DIR}/operators -type d -print -mindepth 1 -maxdepth 1)
 do
     cd "$dirpath"
     operator_name=$(basename ${dirpath})
     echo "* Handling operator ${operator_name} ..."
     
-    podman manifest rm "${operator_name}" || true
-    podman manifest create "${operator_name}"
     build_img="quay.io/stolostron-grc/${operator_name}:latest"
     for arch in amd64 arm64 s390x ppc64le
     do
-        podman build -f Dockerfile --manifest "${operator_name}" \
-          --platform "linux/${arch}" --build-arg "TARGETARCH=${arch}"
+        echo "::group::Building ${operator_name} for ${arch}"
+        ${CONTAINER_TOOL} build -f Dockerfile -t "${build_img}-${arch}" \
+          --platform "linux/${arch}" --build-arg "TARGETARCH=${arch}" \
+          --build-arg "HOSTPLATFORM=${host_platform}" .
+        ${CONTAINER_TOOL} push "${build_img}-${arch}"
+        echo "::endgroup::"
     done
-    podman manifest push "${operator_name}" "${build_img}"
+
+    echo "::group::Building multiarch image and bundle for ${operator_name}"
+    ${CONTAINER_TOOL} manifest rm "${build_img}" || true
+    ${CONTAINER_TOOL} manifest create "${build_img}" "${build_img}-amd64" \
```

Comment:
What's the benefit of doing it this way instead of `manifest add` (or similar) on each loop?